.laminas-ci.json

```diff
@@ -0,0 +1,7 @@
+{
+    "exclude": [
+        {
+            "name": "PHPUnit on PHP 8.1 with lowest dependencies"
```

Comment:
Should lowest deps really be excluded? Shouldn't we bump requirements? :|

Reply:
updated https://github.com/laminas/laminas-skeleton-installer/pull/27/commits/2cd3a68d183e9f7be2006883202f145932761283